hive-site.tf

```diff
@@ -0,0 +1,28 @@
+data "template_file" "hive_site_xml" {
+  template = <<EOF
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<configuration>
+  <property>
+     <name>apiary.path.replacement.enabled</name>
+     <value>true</value>
+  </property>
+%{for alluxio_endpoint in var.alluxio_endpoints}
+%{for s3_bucket in split(",", alluxio_endpoint.s3_buckets)}
+   <property>
+       <name>apiary.path.replacement.regex.alluxio-${s3_bucket}</name>
+       <value>^(s3://)${s3_bucket}/.*</value>
```

Comment:
Your example later down shows:
```
root_url = "alluxio://alluxio1"
```
Since here you are capturing `s3://` and replacing with `alluxio://alluxio1`, what does the final replaced location look like?

E.g., if the original S3 bucket is `s3://apiary-1234-us-west-2-test` and you replace with `"alluxio://alluxio1`, that would result in `"alluxio://alluxio1apiary-1234-us-west-2-test`, which seems like you are missing a delimiter somewhere.

If the `root_url` has requirements to end with some kind of delimiter, that needs to be called out in the comments/docs in here somewhere.

Reply:
correct, with current setup, root_url should include `/` at the end

Reply:
added details in variable comment